project/Dependencies.scala

```diff
@@ -21,7 +21,9 @@ object Dependencies {
 
   val Libraries: Seq[ModuleID] = Seq(
     "com.typesafe.akka" %% "akka-persistence-query" % AkkaVersion,
-    "com.typesafe.slick" %% "slick" % SlickVersion,
+    // Slick 3.5 pulls in slf4j-api 2.2 which doesn't work with Akka
+    ("com.typesafe.slick" %% "slick" % SlickVersion).exclude("org.slf4j", "slf4j-api"),
+    "org.slf4j" % "slf4j-api" % "1.7.36",
```

Comment:
do we need to add the explicit version here? with the exclude in place, I thought it would use what we have defined in upstream Akka dependencies

would be easy to forget this one once we bump slf4j in Akka

Reply:
I was thinking that since we remove the transitive one, we need to make sure there is another one pulled in or else you could end up with a classdef not found in some scenario, users will most likely have the akka-slf4j in there but maybe not always?

Reply:
ok, true, if not using Akka Typed
I'll add a note in the slf4j update ticket